apps/cow-fi/components/SearchBar/index.tsx

```
@@ -0,0 +1,284 @@
+import React, { useState, useEffect, useRef, useTransition } from 'react'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)